demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -87,8 +87,12 @@ def rotate_poses(poses_3d, R, t):
     R = np.array(extrinsics['R'], dtype=np.float32)
     t = np.array(extrinsics['t'], dtype=np.float32)
 
-    frame_provider = InputReader(args.input)
-    is_video = frame_provider.is_video
+    cap = open_images_capture(args.input, args.loop)
+    is_video = isinstance(cap, VideoCapWrapper)
```

Comment:
I reccomend add field `type` (or any) to wrapper classes and use `is_video = cap.type == VIDEO` in app level.

Reply:
Python doesn't need that, `isinstance()` does exactly that.

Reply:
Done

Reply:
Do we need to do anything like that in the first place? Why not use the same logic in all cases?